app/views/layout.slim

```diff
@@ -2,12 +2,24 @@ doctype html
 html
 	head
 		title Tint
+		link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:100,100i,300,300i,400,400i,700,700i,900,900i"
 		link rel="stylesheet" href="/assets/tint.css"
+		link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"
```

Comment:
Oh, you generated an actual icon file?  Ok, then that's the right type.  I just did not anticipate that you had done that.  Most sites just use a PNG named `favicon.ico` these days.  But if you made an icon file that's fine :)